src/lerna/new/index.ts

```diff
@@ -15,6 +16,17 @@ import { default as helpers } from 'handlebars-helpers';
 import { resolve } from 'path';
 import { default as walk } from 'klaw';
 
+/**
+ * Register handlebars helpers.
+ */
+const registerHelpers = (): void => {
+  helpers({ handlebars });
+
+  handlebars.registerHelper('kebabcase', (string: string) => kebabCase(string));
+  handlebars.registerHelper('snakecase', (string: string) => snakeCase(string));
+  handlebars.registerHelper('startcase', (string: string) => startCase(string));
```

Comment:
Alternatively, for a shorter syntax, I think the callbacks could implicitly supply the argument(s):

```js
handlebars.registerHelper('kebabcase', kebabCase);
handlebars.registerHelper('snakecase', snakeCase);
handlebars.registerHelper('startcase', startCase);
```

Reply:
I had tried, but the TS was grumpy.